$HOME/.config/NuGet/nuget.config

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
@praveenkuttappan Any idea why this file is in this PR?

Reply:
Let me check and find how did this get added.